stable/ghost/templates/deployment.yaml

```diff
@@ -49,7 +49,11 @@ spec:
           value: {{ .Values.externalDatabase.host | quote }}
         {{- end }}
         - name: MARIADB_PORT_NUMBER
+        {{- if .Values.mariadb.enabled }}
           value: "3306"
+        {{- else }}
+          value: {{ .Values.externalDatabase.port | default "3306" | quote }}
```

Comment:
The default values should come from **values.yaml**. Therefore, I'd try with sth like:

```
        {{- if and (not .Values.mariadb.enabled) .Values.externalDatabase.port }}
          value: {{ .Values.externalDatabase.port | quote }}
        {{- end }}
```

We don't need to specify **3306** when **".Values.mariadb.enabled"** is true since by default it's the value used when the env. variable **MARIADB_PORT_NUMBER** is not set

Reply:
Ok, I can try that. I did it this way to not change the current behavior of always setting MARIADB_PORT_NUMBER